package.json

```diff
@@ -19,6 +19,7 @@
     "author": "KvNGCzA <akanmuoluwadamilare@yahoo.com>",
     "license": "MIT",
     "dependencies": {
+        "bcrypt": "^3.0.6",
```

Comment:
Why install bcrypt when you have bcryptjs

Reply:
fixed